### DISCUSSION TITLE
Unable to paste images from clipboard in latest release

### DISCUSSION THREAD

**Discussion by chris-sanders** [open-webui/open-webui#7265] [General]
# Bug Report

---

## Installation Method

Installed via Helm Chart

## Environment

Open WebUI: v0.4.3@sha256:bfe5becad8b7a34e3c603a9c5460382e2ab2b83e5cc891ea2dcf9b718c26fe50
Configured with several API keys for OpenAI/Anthropic/etc

- **Browser:** v 1.70.117, sheilds-down no blocking
MacOS 15.1 (24B83)

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ x] I have included the browser console logs.
- [ N/A] I have included the Docker container logs.
- [x ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

In the previous release I could copy an image from clipboard and paste it into the text field and it was passed through to the model. 

With the latest update pasting doesn't work, I can't find any logs, errors, or messages. I simply can't paste after my auto update to the above tag.

Console logs shows: `RichTextInput.svelte:138 [tiptap warn]: Duplicate extension names found: ['codeBlock']. This can lead to issues.` but nothing else, no logging when pasting fails.

I can definitively say that I had a browser window open that hadn't re-authenticated and I was able to paste into the text area. After I tried to submit the session was noticed to be logged out, refreshed, and running the new version. At this point pasting didn't work, which I think is pretty definitive to be a UI change in the latest release.

## Reproduction Details

Install the latest version, copy an image to clip board and ctrl-v, right-click paste, or similar to see it doesn't paste.

## Logs and Screenshots

There are no logs generated in the container, or in the developer console.
I would include a screenshot but it's just a screenshot of the default landing page.

**Comment by tjbck** [comment]
![image](https://github.com/user-attachments/assets/f3327818-46cb-4caa-8352-c24e903cf9fe)

Definitely works with Chrome/Safari/FF on MacOS. Perhaps this is a browser specific issue? More details would be helpful here!

**Comment by chris-sanders** [comment]
Just to check I just copied the image you have there, and that worked fine.
I went back to my original source, and it doesn't work but that image was out of Google Docs.
I went to the source image from the GDoc and that one works.

Maybe there's a change in image formats or size of image at play? I'm a little lost how to find out what an image in the clipboard looks like as far as file type or size. While my original issue does still seem to exist, it also seems to be an edge case of some sort. I'm fine closing this, I don't honestly know how to dig deeper I'll just avoid copying images out of Google Docs?

**Comment by tjbck** [comment]
Hmm, definitely on our radar now. Appreciate the issue report!
